keras/engine/topology.py

```diff
@@ -2832,26 +2848,24 @@ def preprocess_weights_for_loading(layer, weights,
                                                     (2, 3, 1, 0))
                 weights = [kernel, recurrent_kernel, bias]
 
-    if original_backend and K.backend() != original_backend:
-        conv_layers = ['Conv1D',
-                       'Conv2D',
-                       'Conv3D',
-                       'Conv2DTranspose']
-        if layer.__class__.__name__ in conv_layers:
-            weights[0] = conv_utils.convert_kernel(weights[0])
-        if layer.__class__.__name__ == 'ConvLSTM2D':
-            weights[0] = conv_utils.convert_kernel(weights[0])
-            weights[1] = conv_utils.convert_kernel(weights[1])
+        if original_backend and K.backend() != original_backend:
+            conv_layers = ['Conv1D',
+                           'Conv2D',
+                           'Conv3D',
+                           'Conv2DTranspose']
+            if layer.__class__.__name__ in conv_layers:
+                weights[0] = conv_utils.convert_kernel(weights[0])
+            if layer.__class__.__name__ == 'ConvLSTM2D':
+                weights[0] = conv_utils.convert_kernel(weights[0])
+                weights[1] = conv_utils.convert_kernel(weights[1])
```

Comment:
I don't understand the reindentation here. The initial indentation was the correct one.